<dec f='include/boost/exception/exception.hpp' l='187' type='void boost::exception_detail::copy_boost_exception(boost::exception * , const boost::exception * )'/>
<dec f='include/boost/exception/exception.hpp' l='261' type='void boost::exception_detail::copy_boost_exception(boost::exception * , const boost::exception * )'/>
<def f='include/boost/exception/exception.hpp' l='410' ll='421' type='void boost::exception_detail::copy_boost_exception(boost::exception * a, const boost::exception * b)'/>
